src/components/BoardCell/BoardCell.vue

```diff
@@ -0,0 +1,30 @@
+<script setup lang="ts">
+defineProps<{
+  background: "light" | "dark";
+  position: string;
+}>();
```

Comment:
I am unsure if this is the correct way to define the props on a component in Vue with Typescript. May need to look at the docs to ensure that I am not mixing Options and Composition syntax 🤔